Integrate django-environ settings

PetrushynskyiOleksii opened the pull request:
**Close #20**

Also add instructions to readme:
- How to run site locally
- How to deply on Heroku servise

PetrushynskyiOleksii committed:
Integrate django-environ into project

PetrushynskyiOleksii committed:
Add django-environ to requirements

PetrushynskyiOleksii committed:
Add instruction to readme

- How to run locally site
- How to deploy on Heroku servise

webknjaz commented on README.md:
```diff
@@ -1,3 +1,30 @@
 
-# Django-girls-blog-PetrushynskyiOleksii
+### Installation
+- [Locally](https://github.com/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii#Lokaly)
```
Actually you could omit URL part and make the target relative (as easy as `#Lokaly`)

webknjaz replied:
Oh, btw it doesn't work now because of a typo

webknjaz commented on README.md:
```diff
@@ -1,3 +1,30 @@
+- [Locally](https://github.com/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii#Lokaly)
+- [Heroku](https://github.com/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii#Heroku)
+
+##### Locally:  
```
What about `.env`?

webknjaz commented on README.md:
```diff
@@ -1,3 +1,30 @@
+1. Register account on [Heroku](https://www.heroku.com/)
+2. Install [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) locally
+3. Login to Heroku : `heroku login`
+4. Create name of your site : `heroku create <name-of-site>`
```
how to auth with ssh?

webknjaz commented on README.md:
```diff
@@ -1,3 +1,30 @@
+6. Start web process : `heroku ps:scale web=1`
+7. Run migrate and create superuser : `heroku run python manage.py migrate`
+
+Also you can create superuser for admin page : `python manage.py createsuperuser`
```
This is inconsistent

webknjaz commented on mysite/settings.py:
```diff
@@ -1,23 +1,21 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '_u)sp0-q^_c!s645c-g5=whl97qsm&wxnw6rajs(hhri93wi9s'
+SECRET_KEY = env('SECRET_KEY', default='_u)sp0-q^_c!s645c-g5=whl97qsm&wxnw6rajs(hhri93wi9s')
```
it's not in example. it's better to not have a default (except for .env example), because one MUST create smth not tracked in git for their prod. it's a security issue otherwise.

webknjaz reviewed (changes requested):
leaking security and docs.
after fixing, try to set-up this project in a fresh env using **ONLY** instructions written in README.

PetrushynskyiOleksii committed:
Fix settings for django environ

  *Delete default value for secret key
  *Add .env
    - .env contains value of secret key, debug and django settings
       module
  *Add example of .env file

webknjaz commented on mysite/.env.example:
```diff
@@ -1 +0,0 @@
-DEBUG=on
```
You may leave the example file in repo

webknjaz commented on README.md:
```diff
@@ -1,3 +1,36 @@
+6. Create a new virtualenv : `pyenv virtualenv 3.6.4 <name of virtualenv>`.
+7. Activate virtualenv : `pyenv local <name of virtualenv>`.
+8. Install requirements for a project : `pip install -r requirements.txt`
+9. Add .env in mysite folder, like in .env.example.
```
You should return example file (and list some secret there)

PetrushynskyiOleksii replied:
@webknjaz done

PetrushynskyiOleksii committed:
Add secret key to travis config
